libs/neo4j/langchain_neo4j/chains/graph_qa/cypher.py

```diff
@@ -62,6 +64,23 @@ def extract_cypher(text: str) -> str:
     return matches[0] if matches else text
 
 
+def extract_filter(text: str) -> str:
```

Comment:
Could you please add a unit test for this under libs/neo4j/tests/unit_tests/chains/test_graph_qa.py?